README.md

```diff
@@ -82,6 +84,9 @@ resources relevant to the project.
 
 If you're looking for something to work on, check out the [help wanted](https://github.com/desktop/desktop/issues?q=is%3Aissue+is%3Aopen+label%3A%22help%20wanted%22) label.
 
+# Purpose
+The purpose of Github Destop is that, it allows you to collaboarate with your team, using some of the best practices. 
+
```

Comment:
A purpose section like what you're suggesting is probably better suited on a marketing page like `desktop.github.com`. Since this README is primarily aimed for contributors I think we should keep it focused on that.

```suggestion
```

Reply:
True